src/app/interfaces/api-event-directory.interface.ts

```diff
@@ -17,4 +17,5 @@ export type ApiEventDirectory = {
   'truecommand.config': { response: TrueCommandConfig };
   'vm.query': { response: VirtualMachine };
   'zfs.pool.scan': { response: ResilverData };
+  'support.attach_ticket': { response: Job };
```

Comment:
This file is only used with `subscribe` method of `WebsocketService`. Doesn't seem to be needed.